arbitrum-docs/how-arbitrum-works/06-optimistic-rollup.mdx

```diff
@@ -0,0 +1,272 @@
+
+### Rules for Confirming or Rejecting RBlocks
+
+The rules for resolving RBlocks are fairly simple.
+
+The first unresolved RBlock can be confirmed if:
+
+- the RBlock's predecessor is the latest confirmed RBlock, and
+- the RBlock's deadline has passed, and
+- there is at least one bonder, and
+- All bonders are bonded to the RBlock.
+
+The first unresolved RBlock can be rejected if:
+
+- the RBlock's predecessor has been rejected, or
+- all of the following are true:
+  - the RBlock's deadline has passed, and
+  - there is at least one bonder, and
+  - no bonder is bonded on the RBlock.
+
+A consequence of these rules is that once the first unresolved RBlock's deadline has passed (and assuming there is at least one bonder bonded on something other than the latest confirmed RBlock), the only way the RBlock can be unresolvable is if at least one bonder is bonded on it and at least one bonder is bonded on a different RBlock with the same predecessor. If this happens, the two bonders are disagreeing about which RBlock is correct. It’s time for a challenge, to resolve the disagreement.
+
+## Delays
+
+Even if the Assertion Tree has multiple conflicting RBlocks and, say, multiple disputes are in progress, validators can continue making assertions; honest validators will simply build on the one valid RBlock (intuitively: an assertion is also an implicit claim of the validity of all of its parent-assertions.) Likewise, users can continue transacting on L2, since transactions continue to be posted in the chain's inbox.
+
+The only delay that users experience during a dispute is of their [L2 to L1 messages](/how-arbitrum-works/11-l2-to-l1-messaging.mdx) (i.e., "their withdrawals"). Note that a "delay attacker" who seeks to grief the system by deliberately causing such delays will find this attack quite costly, since each bit of delay-time gained requires the attacker lose another stake.
+
+### Detailed Spec
+
+For a more detailed breakdown / specification of the assertion tree protocol, see [Inside Arbitrum](/how-arbitrum-works/assertion-tree.mdx).
+
+## Validators
+
+Arbitrum full nodes normally "live at Layer 2" which means that they don’t worry about the rollup protocol but simply treat their Arbitrum chain as a mechanism that feeds inbox messages to the State Transition Function to evolve the Layer 2 chain and produce outputs.
+
+Some Arbitrum nodes will choose to act as _validators_. This means that they watch the progress of the rollup protocol and participate in that protocol to advance the state of the chain securely.
+
+Not all nodes will choose to do this. Because the rollup protocol doesn’t decide what the chain will do but merely confirms the correct behavior that is fully determined by the inbox messages, a node can ignore the rollup protocol and simply compute for itself the correct behavior.
```

Comment:
```suggestion
Not all nodes will choose to do this. Because the Rollup protocol doesn’t decide what the chain will do but merely confirms the correct behavior that is fully determined by the inbox messages, a node can ignore the rollup protocol and simply compute for itself the correct behavior.
```